### ISSUE TITLE
can we launch spells?

### ISSUE THREAD

**Issue by kurokirasama** [asmfreak/habitipy#16]
if not, it would be nice to add it :D

**Comment by ASMfreaK** [comment]
yes we can, it's a bug that we can't from cli api.

**Comment by hardaker** [comment]
FYI, I've implemented spell casting in my branch: github.com:hardaker/habitipy

I'm waiting to submit a PR until after some of my other PRs get accepted otherwise they'll be conflicts.

**Comment by asmfreak** [comment]
@hardaker You can submit all your work in separate PRs (and in separate branches). I'll merge and rebase them as needed.

**Comment by hardaker** [comment]
they all have similar code areas so branches will conflict if they're not done in the right order, that's why I haven't done just that.

**Comment by hardaker** [comment]
True.  I in general haven't done that for enhancements as it should be up to the maintainer to decide if the added support is "enough" to warrant closing a feature request.  Feature requests often can span multiple requests.

[having said that, I think this one does warrant closing but I should have said in the message that it's possible someone would consider my added support not complete]
